pkg/kopia/command/storage/azure_args.go

```diff
@@ -0,0 +1,55 @@
+package storage
+
+import (
+	"github.com/Azure/go-autorest/autorest/azure"
+	"github.com/kanisterio/kanister/pkg/logsafe"
+	"github.com/kanisterio/kanister/pkg/secrets"
+	"github.com/pkg/errors"
+	v1 "k8s.io/api/core/v1"
+)
+
+const (
+	azureSubCommand         = "azure"
+	azureContainerFlag      = "--container"
+	azurePrefixFlag         = "--prefix"
+	azureStorageAccountFlag = "--storage-account"
+	azureStorageKeyFlag     = "--storage-key"
+	azureStorageDomainFlag  = "--storage-domain"
+)
+
+func kopiaAzureArgs(locationSecret, locationCredSecret *v1.Secret, artifactPrefix string) (logsafe.Cmd, error) {
```

Comment:
All these helpers would have to be refactored to read from files. You could add a util library that reads from files based on the storage type.